privacy-policy/index.html

```diff
@@ -61,21 +60,24 @@ <h3 id="accesstoannotations">Access to annotation data</h3>
 
 <p>With respect to Hypothesis staff, a small number of engineering staff have database access that lets them see any annotation or profile data in your account, in order to operate the service and address problems with the system.  Staff is not allowed to re-disclose private user information.</p>
 
-
-<h3 id="permissions">Chrome extension permissions</h3>
+<h2 id="permissions">Chrome Extension Permissions</h2>
 
 <p>The Hypothesis extension for Chrome will ask users to grant certain permissions at the time of installation. The extension needs them in order to work with the user’s browser and the websites the user visits. More detailed information about how we use these permissions on this blog post:  <a href="https://hypothes.is/blog/hypothes-is-chrome-extension-permissions/" target='_blank'>https://hypothes.is/blog/hypothes-is-chrome-extension-permissions/</a></p>
 
-<h3 id="contact">Contact with users</h3>
+<h2 id="contact">Contact with Users</h2>
 
 <p>We send account registration, activation and password reset notifications via email. We also send event notifications such as about replies to annotations via email, with the option to opt-out.  Approximately every quarter we send out a newsletter, “The Margin”, with the option to opt-out.  In the future we may also send information about important service updates or substantial changes in Privacy Policy or our Terms of Service.</p>
 
-<h3 id="consent">Consent</h3>
+<h2 id="breaches">Breaches of Privacy</h2>
+
+<p>In case of a breach of private information, we will notify users within 24 hours.</p>
```

Comment:
Not sure this wording covers us correctly. 24 hours from when the breach happens? 24 hours from when we become aware of the breach? Notify users how?